app/assets/stylesheets/modules/_family.scss

```diff
@@ -128,6 +124,83 @@
       background-size: 100%;
       display: inline-block;
       height: em(210);
+      margin-top: em(40);
+  }
+}
+
+.family-names {
+  text-align: center;
+  background-color: tint($light-gray, 60);
+  border-radius: $base-border-radius;
+  margin: em(10) em(0) em(30);
+  padding: em(20) 0;
+
+  @include media(medium-screen-up){
+    margin: em(10) em(80) em(30);
+    padding: em(20) 0;
+  }
+}
+
+.family-names-item {
+  color: tint($base-font-color, 40);
+  font-family: "Courier";
+  font-size: em(12);
+  height: em(25);
+  margin: 0 auto;
+  overflow: hidden;
+  white-space: nowrap;
+
+  @include media(medium-screen-up){
+    font-size: em(20);
+  }
+}
+
+.family-names-item:nth-child(1) {
+  animation: type 9s steps(37, end);
+  animation-iteration-count: infinite;
```

Comment:
Properties should be ordered animation, -webkit-animation-delay, animation-iteration-count, border-right, margin-bottom, width